README.md

```diff
@@ -84,6 +85,16 @@ Then you can enable the extension with the *Gnome Tweak Tool*, the *Extensions*
 gnome-extensions enable flypie@schneegans.github.com
 ```
 
+With Fly-Pie 4 (currently in development), you will be able to install the extension with a simpler command.
+
+<!-- These are the instructions for version 4.
+Extract the `zip` archive. Open the folder in a terminal window and execute this command:
+
+```bash
+./create-release -i
```

Comment:
At this point the user downloaded a release from Github. Or we could actually provide a terminal command for downloading the release. I think the correct command would be something like this:
```suggestion
wget https://github.com/Schneegans/Fly-Pie/releases/latest/download/flypie@schneegans.github.com.zip
gnome-extensions install flypie@schneegans.github.com.zip --force
```

Reply:
Yes, the command works for me.

Reply:
Should we really use `force` here? It overwrites existing installs of the extension. I think in most cases, the extension isn't installed when running this command. Or we could provide it like this:
```suggestion
wget https://github.com/Schneegans/Fly-Pie/releases/latest/download/flypie@schneegans.github.com.zip
gnome-extensions install flypie@schneegans.github.com.zip
If you have Fly-Pie already installed and want to upgrade it to the latest version, append the `--force` flag to the last command to overwrite existing installs of Fly-Pie.
```

Reply:
Sounds good!